django/db/backends/base/features.py

```diff
@@ -227,6 +227,12 @@ class BaseDatabaseFeatures(object):
     # Place FOR UPDATE right after FROM clause. Used on MSSQL.
     for_update_after_from = False
 
+    # Combinatorial flags
+    supports_select_union = True
```

Comment:
I was looking for the lowest common denominator, but have no problem to switch that to true. I'd leave `supports_slicing_ordering_in_compound` as False though since most databases do not support it and afaik it is an extension to the standard.